leap.js

```diff
@@ -23,7 +23,7 @@ function Year(year) {
  * Whether given year is a leap year.
  */
 Year.prototype.isLeap = function () {
- if (((year%100 ==0) && (year%400 ==0))|| (year%4 ==0)){
+ if ((this.year%100 ==0) && (this.year%400 ==0) || (this.year%4 ==0)){
     return "leap year";
   }
```

Comment:
You miss the `else` statement..
Remind that your function have to return a `boolean` value :smile: !